src/components/Draft.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import { ListGroup } from 'react-bootstrap';
+
+/** Stateless React component that displays a set of predictions. */
+const Draft = ({ draft, queens }) => {
+  const sortedPredictionList = draft
+    .filter((entry) => entry.placement)
+    .sort((a, b) => b.placement - a.placement)
+    .map((entry) => queens.find((queen) => queen.id === entry.queen_id));
+
+  const listGroupItems = sortedPredictionList.map((queen) => (
+    <ListGroup.Item className='prediction-item' key={queen.id}>
+      {queen.name}
+    </ListGroup.Item>
+  ));
+
+  return <ListGroup horizontal>{listGroupItems}</ListGroup>;
+};
+
+const queenShape = {
```

Comment:
These "shapes" should probably be pulled out into their own file at some point for global use

Reply:
Good call! Issue created here: https://trello.com/c/sAqDnDy3/14-create-shared-file-for-data-shapes